src/Data/Fin/Properties.agda

```diff
@@ -570,6 +587,76 @@ combine-remQuot {suc n} k i with splitAt k i | P.inspect (splitAt k) i
   i                                        ∎
   where open ≡-Reasoning
 
+toℕ-combine : ∀ {n k} (x : Fin n) (y : Fin k) → toℕ (combine x y) ≡ k ℕ.* toℕ x ℕ.+ toℕ y
```

Comment:
In this proof and the ones below, we generally use `m` and `n` for naturals and `i`,`j`,`k` for `Fin` types rather than `n` `k` `x` and `y`.